docker/testenv-ingestd/Dockerfile

```diff
@@ -0,0 +1,16 @@
+ARG LSST_STACK_VERSION
+FROM lsstsqre/centos:${LSST_STACK_VERSION}
+
+USER root
+RUN yum install -y nc
```

Comment:
Can you add a comment explaining why this is needed?

Reply:
This install of "nc" should have been removed.   I was using it to debug something early on, and never took it out.   I'll remove it.